src/core/background/scrobbler/maloja-scrobbler.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+/**
+ * Module for communication with Maloja
+ */
+define((require) => {
+	const Util = require('util/util');
+	const BaseScrobbler = require('scrobbler/base-scrobbler');
+	const ServiceCallResult = require('object/service-call-result');
+
+	class Maloja extends BaseScrobbler {
+
+		/** @override */
+		getStorageName() {
+			return 'Maloja';
+		}
+
+		/** @override */
+		getId() {
+			return 'maloja';
+		}
+
+		/** @override */
+		getLabel() {
+			return 'Maloja';
+		}
+
+		/** @override */
+		getStatusUrl() {
+			return null;
+		}
+
+		/** @override */
+		getUsedDefinedProperties() {
+			return ['userApiUrl', 'userToken'];
+		}
+
+		/** @override */
+		getProfileUrl() {
+			return null;
+		}
+
+		/** @override */
+		async getAuthUrl() {
+			return null;
+		}
+
+		/** @override */
+		async getSession() {
+			return { sessionID: this.userToken };
```

Comment:
This makes the extension to think a user is always signed in to Maloja. I suggest to check if the `userToken` property is not empty before returning the session info; throw an error otherwise.